### PR TITLE
Update Bracken data manager

### PR DESCRIPTION
**Pull request by gallardoalba**
FOR CONTRIBUTOR:
* [x] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [x] - License permits unrestricted use (educational + commercial)
* [ ] - This PR adds a new tool or tool collection
* [ ] - This PR updates an existing tool or tool collection
* [x] - This PR does something else (explain below)

I think that it fixes a problem related to the Bracken databases: [history](https://usegalaxy.eu/u/gallardoalba/h/bracken-update-history).

### CODE REVIEW

**Comment by gallardoalba** [comment]
I have tested it locally, but not sure how to include a new test in the data manager itself.

File: data_managers/data_manager_build_bracken_database/data_manager_conf.xml [hunk 1]
```diff
@@ -4,7 +4,14 @@
             <output>
                 <column name="value"/>
                 <column name="name"/>
-                <column name="path" output_ref="out_file"/>
+                <column name="path" output_ref="out_file">
+                    <move type="directory">
+                        <source>${path}</source>
+                        <target base="${GALAXY_DATA_MANAGER_DATA_PATH}">bracken_databases/${path}</target>
+                    s</move>
```
Comment (bgruening) [inline data_managers/data_manager_build_bracken_database/data_manager_conf.xml:11]: there is a `s`

**Comment by bgruening** [comment]
@dfornika can you maybe look at this?

File: data_managers/data_manager_build_bracken_database/data_manager_conf.xml (hunk 1 above)
Comment (gallardoalba) [reply data_managers/data_manager_build_bracken_database/data_manager_conf.xml]: Thanks!

**Comment by gallardoalba** [comment]
I'm not sure how to fix the test, perhaps you have an idea @EngyNasr?

**Comment by gallardoalba** [comment]
Thanks a lot @blankenberg, I was struggling with it.

**Comment by blankenberg** [comment]
My pleasure @gallardoalba. Mind doing a test install to confirm that it is actually working?

**Comment by gallardoalba** [comment]
> My pleasure @gallardoalba. Mind doing a test install to confirm that it is actually working?

Sure!

**Comment by gallardoalba** [comment]
> My pleasure @gallardoalba. Mind doing a test install to confirm that it is actually working?

Hi @blankenberg, sorry for the delay. Unfortunately, it is still failing:

Screenshot from 2022-04-07 23-11-09

**Comment by bernt-matthias** [comment]
Can this be bumped to 2.7?

**Comment by bernt-matthias** [comment]
Is https://github.com/galaxyproject/tools-iuc/issues/4141 fixed here?

**Comment by gallardoalba** [comment]
Not, it is still not working.

**Comment by bernt-matthias** [comment]
What is the state here? Tests seem to pass?

**Comment by gallardoalba** [comment]
> What is the state here? Tests seem to pass?

I'll check it again tomorrow, but last time I tested in my local instance and I found still some errors.

**Comment by bernt-matthias** [comment]
Old but gold? Is this PR still needed?

**Comment by pimarin** [comment]
The bracken nomenclature is a problem into the datamanager.
I experienced it to standardize a workflow.
The problem; I use kraken2 and bracken which deal with closed database but bracken need the database with kmer information so 2 data managers.
I can submit by a logical input the name of database in kraken2 to be used :
`k2_pluspf_16gb_20210517` because the script use :
```
    database_value = "_".join([
        now,
        "standard",
        "kmer-len", str(kraken2_args["kmer_len"]),
        "minimizer-len", str(kraken2_args["minimizer_len"]),
        "minimizer-spaces", str(kraken2_args["minimizer_spaces"]),
        "load-factor", str(kraken2_args["load_factor"]),
```
But for bracken if you use the equivalent input provided by your galaxy web server the tool failed with an error of database name : 
```
parameter 'kmer_distr': an invalid option ('2023-08-24T153645Z_standard_prebuilt_pluspf_16gb_2022-06-07') was selected (valid options: bf6177af-7f5b-4005-9bec-fc7c89c3cb1c,41c59a90-3759-486c-b865-bcffc966adde) Using default: '41c59a90-3759-486c-b865-bcffc966adde'.
```
Because bracken  use the uuid of the file :
`database_value = str(uuid.uuid4())` 
as : `41c59a90-3759-486c-b865-bcffc966adde`
But this information is not available by a simple user if he wants to use a workflow from a public repository

**Pull request closed without merging**